packages/popover/src/Popover.tsx

```diff
@@ -93,24 +93,29 @@ function Popover({
 
   const viewportSize = useViewportSize();
 
+  // We already recalculate the position when the element becomes active,
+  // So it's safe for us to disable the mutation observers when the element isn't active.
+  const observeMutations = adjustOnMutation && active;
```

Comment:
So we only want to run the callback when the Popover is active? I feel like that's at odds with what you wrote?

Reply:
How so? The idea here is to avoid unnecessary mutation observers on the page. Previously, there was no logic to prevent a mutationObserver from being set when the Popover was inactive.

Reply:
Ah I think I got confused by "We already recalculate the position" which made it sound like we don't want to do it again. But we do? Dunno, can ignore me, just reads a little confusing to me.

Reply:
When the Tooltip becomes active, we calculate the desired position initially, so we don't need the mutation observer prior to it becoming active. I can try to make this comment more clear.